package.json

```diff
@@ -24,14 +24,15 @@
   "peerDependencies": {
     "babel-eslint": "^10.0.1",
     "eslint": "^5.12.0",
```

Comment:
Do we need to bump this too, in order to support the new version changes you made below?

Reply:
no, v18 works with 5 | 6

Reply:
Awesome! maybe something for the future we can look at updating 😄